modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -188,20 +191,139 @@
                   }
                 }
 
-      describe 'Getting a successful response' do
-        response '200', 'upload response' do
-          it 'returns a valid 200 response' do
-            one = 1
-            expect(one).to eq(1)
+      describe 'Getting an accepted response' do
+        response '202', 'upload response' do
+          let(:data) do
+            temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                   'disability_compensation', 'form_526_json_api.json').read
+            temp = JSON.parse(temp)
+
+            temp
+          end
+
+          let(:scopes) { %w[system/claim.write] }
+          let(:auto_claim) { create(:auto_established_claim) }
+          let(:attachment1) do
+            Rack::Test::UploadedFile.new(Rails.root.join(*'/modules/claims_api/spec/fixtures/extras.pdf'.split('/'))
+                                                     .to_s)
+          end
+          let(:attachment2) do
+            Rack::Test::UploadedFile.new(Rails.root.join(*'/modules/claims_api/spec/fixtures/extras.pdf'.split('/'))
+                                                     .to_s)
+          end
+          let(:id) { auto_claim.id }
+
+          before do |example|
+            stub_poa_verification
+            stub_mpi
+
+            mock_ccg(scopes) do
+              VCR.use_cassette('claims_api/disability_comp') do
+                submit_request(example.metadata)
+              end
+            end
+          end
+
+          after do |example|
+            example.metadata[:response][:content] = {
+              'application/json' => {
+                example: JSON.parse(response.body, symbolize_names: true)
+              }
+            }
+          end
```

Comment:
I am curious if the example is correct.  In v1 it appears to be the file upload as an example but this looks like a 526 form submission?  
<img width="873" alt="Screenshot 2023-11-17 at 9 32 45 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/5e2c1b93-743c-4de9-aaef-bcc82cf64613">